plugins/module_utils/sops.py

```diff
@@ -91,7 +91,7 @@ def f(value, arguments_pre, arguments_post, env, version):
         env[argument_name] = value
 
     return f
-
+ 
```

Comment:
This line still has a change that sholdn't be there. This is also what pylint and pep8 complain about: you added a newline to an otherwise empty line.

Reply:
Looks to me like the space is the problem, not the newline. Removed the space in the newest commit.

Reply:
Ah, I wanted to write "space", not "newline" :)